data/types/basal/test/suppressed.go

```diff
@@ -1,13 +1,62 @@
 package test
 
 import (
+	"github.com/tidepool-org/platform/data"
+	testData "github.com/tidepool-org/platform/data/test"
 	"github.com/tidepool-org/platform/data/types/basal"
-
-	// Pull in init function to ensure test environment
-	_ "github.com/tidepool-org/platform/test"
+	"github.com/tidepool-org/platform/pointer"
+	"github.com/tidepool-org/platform/test"
 )
 
-func NewSuppressed(typ interface{}, deliveryType interface{}, annotations *[]map[string]interface{}, rate interface{}, scheduleName interface{}, suppressed *basal.Suppressed) *basal.Suppressed {
+func NewSuppressedScheduled() *basal.Suppressed {
+	datum := basal.NewSuppressed()
+	datum.Type = pointer.String("basal")
+	datum.DeliveryType = pointer.String("scheduled")
+	datum.Annotations = testData.NewBlobArray()
+	datum.Rate = pointer.Float64(test.RandomFloat64FromRange(basal.RateMinimum, basal.RateMaximum))
+	datum.ScheduleName = pointer.String(NewScheduleName())
+	return datum
+}
+
+func NewSuppressedSuspend() *basal.Suppressed { // Not legal, but used for negative unit tests
```

Comment:
should it live with the tests then??

Reply:
It is with the tests. Any files in any `test` directory is guaranteed to NOT be included in the executable binary, only in the tests.

Reply:
ha ha - I didn't spot that :)